### PR TITLE
Feature/show more button

### PR DESCRIPTION
**Pull request by salesfelipe**
#### What is the purpose of this pull request?
Implement another option for loading more products

#### How should this be manually tested?
[workspace](https://felipe--storecomponents.myvtex.com/eletronicos/d)

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [x] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/index.js [hunk 1]
```diff
@@ -1,66 +1,225 @@
...
+    const isLoading = loading || this.props.loading
+    const to = (page - 1) * maxItemsPerPage + products.length
+
+    console.log('Search Container')
+
     return (
-      <SearchResultInfiniteScroll
-        {...this.props}
+      <SearchResult
+        breadcrumbsProps={breadcrumbsProps}
```
Comment (andreldsa) [inline react/index.js]: Pass all the props

File: react/index.js [hunk 2]
```diff
@@ -1,112 +1,292 @@
...
+      brands: Brands,
+      specificationFilters: SpecificationFilters,
+      priceRanges: PriceRanges,
+      priceRange: priceRange,
+      hiddenFacets,
+      tree: CategoriesTrees
+    }
+
+    return (
+      <PopupProvider>
```
Comment (brunojdo) [inline react/index.js]: What does this component do?

File: react/index.js [hunk 3]
```diff
@@ -1,112 +1,292 @@
...
-
-  render() {
-    return <SearchResultInfiniteScroll {...this.props} />
+      summary: {
+        title: 'editor.search-result.summary.title',
+        type: 'object',
+        properties: ProductSummary.getSchema(props).properties,
+      },
+      showMore: {
+        type: 'boolean',
```
Comment (lucasecdb) [inline react/index.js]: Change to be an enum like a `type`, that can have several option (infinite scroll, pagination or show more button)

File: react/global.css [hunk 4]
```diff
@@ -3,7 +3,8 @@
   grid-template-columns: 267px repeat(5, auto);
   grid-template-areas:
     ". breadcrumb . quantity orderby limit"
-    "filters gallery gallery gallery gallery gallery";
+    "filters gallery gallery gallery  gallery gallery"
+    ". . . button . .";
```
Comment (lucasecdb) [inline react/global.css]: Could be `". pagination pagination pagination pagination pagination"`

File: react/index.js (hunk 2 above)
Comment (salesfelipe) [reply react/index.js]: This is the menu displayed on mobile

File: react/index.js (hunk 3 above)
Comment (salesfelipe) [reply react/index.js]: 👍

**Pull request merged**